detox/src/devices/drivers/android/AndroidDriver.test.js

```diff
@@ -136,41 +155,191 @@ describe('Android driver', () => {
     });
   });
 
-  describe('Notification data handling (user-notification-data-URL arg)', () => {
-    const launchArgs = Object.freeze({
+  describe('URL runtime delivery handling', () => {
```

Comment:
Now coverage to existing api's (i.e. launch with url & sendURL() flow) - but aligned with a modified implementation (as previously mentioned).